apps/storybook-e2e/src/e2e/sections/cookie-consent/vl-cookie-consent.stories.cy.ts

```diff
@@ -1,38 +1,16 @@
 const cookieConsentUrl =
     'http://localhost:8080/iframe.html?id=sections-cookie-consent--cookie-consent-default&viewMode=story';
 
-describe('story vl-cookie-consent', () => {
-    it('should contain the `Cookie-toestemming`', () => {
+describe('story vl-cookie-consent - default', () => {
+    it('should display story', () => {
         cy.visit(cookieConsentUrl);
-
-        cy.get('button#button-open-cookie-consent').click();
-        cy.get('vl-cookie-consent').shadow().find('vl-modal').shadow().find('h2').contains('Cookie-toestemming');
-        cy.get('vl-cookie-consent')
-            .shadow()
-            .find('vl-modal')
-            .find('.vl-button')
-            .scrollIntoView()
-            .contains('Ik begrijp het')
-            .click();
-    });
-
-    it('should contain the given matomo id & matomo url', () => {
-        const matomoId = 12345;
-        const matomoUrl = 'fake-matomo-url';
-
-        cy.visit(`${cookieConsentUrl}&args=matomoId:${matomoId};matomoUrl:${matomoUrl}`);
-
-        cy.get('script#vl-cookie-consent-matomo-script')
-            .should('contain.text', `_paq.push(['setSiteId', ${matomoId}]);`)
-            .should('contain.text', `var u='${matomoUrl}'`);
+        cy.get('vl-cookie-consent');
     });
 
-    it('should emit event on close', () => {
+    it('should contain the `Cookie-toestemming`', () => {
```

Comment:
Deze test kan ook geplaatst worden bij de component tests van dit component.

Reply:
het bestaat ook in de component testen `vl-cookie-consent.section.cy.ts`,  maar in de storybook om de `cookie consent` echt te krijgen moet ik de btn in de story op klikken. Dus ik test gewoon dat als je de btn op klikt zie je de story, maar alles qua functionaliteit is getest binnen `vl-cookie-consent.section.cy.ts`